src/Symfony/Component/Notifier/Channel/ChannelPolicy.php

```diff
@@ -18,11 +18,8 @@
  */
 final class ChannelPolicy implements ChannelPolicyInterface
 {
-    private array $policy;
-
-    public function __construct(array $policy)
+    public function __construct(private array $policy)
```

Comment:
as noted in another similar PR, let's put CPP on their own lines
please review all similar PRs please

Reply:
Done